backend/node/genvm/base.py

```diff
@@ -222,24 +235,45 @@ async def run_contract(
                 else:
                     function_to_run(*args)
             except Exception as e:
-                print("\nError running contract", e)
-                print(f"\n{traceback.format_exc()}")
+                trace = traceback.format_exc()
                 error = e
+                print("Error executing method", error)
```

Comment:
@denishacquin where are these logs getting printed?

Reply:
In the terminal. The intention here was merely to print the error and trace directly to the terminal when it happens but it might be redundant with the "pretty" that happen below.

Reply:
Nevermind, they actually only get sent back with the send_message.

How about we print the stdout directly to the terminal if it exists (for clarity of debugging backend via the terminal) and keep sending to the log event bus like so:
<img width="746" alt="image" src="https://github.com/user-attachments/assets/21138ba0-8ce0-443c-8101-54bd9b0be3b1">

Resulting in this in terminal (ok vs error):
<img width="971" alt="image" src="https://github.com/user-attachments/assets/cf339aeb-c36d-4399-9ed2-7a0a9300321a">
<img width="959" alt="Screenshot 2024-09-27 at 12 23 42" src="https://github.com/user-attachments/assets/b3717f94-7066-4a4e-9c3e-5125bf3c4f3b">

And this in frontend (ok vs error):
<img width="509" alt="image" src="https://github.com/user-attachments/assets/331ce29b-9fed-49dd-aeaf-39dc33b9a804">
<img width="896" alt="image" src="https://github.com/user-attachments/assets/c6ed9451-87b3-4dd9-8cd7-cafc4bc9140e">

Also added a check to only print the "OK" message when execution is successful.

Reply:
OK I think that is ok to also print the errors in the terminal for a better debugging